### PR TITLE
Remove video_translation_languages field from course_run object. 

### PR DESCRIPTION
**Pull request by schenedx**
The field name is not accurate and the data is not being used.
@clintonb Please help review
Will create the PR for the migration file after this is merged.

### CODE REVIEW

File: course_discovery/apps/course_metadata/models.py [hunk 1]
```diff
@@ -406,8 +406,6 @@ class CourseRun(TimeStampedModel):
     syllabus = models.ForeignKey(SyllabusItem, default=None, null=True, blank=True)
     card_image_url = models.URLField(null=True, blank=True)
     video = models.ForeignKey(Video, default=None, null=True, blank=True)
-    video_translation_languages = models.ManyToManyField(
```
Comment (clintonb) [inline course_discovery/apps/course_metadata/models.py]: You can’t get rid of the field, yet. That comes in the second/last PR.

**Review by clintonb — CHANGES_REQUESTED** [review]
Revert the model change.

File: course_discovery/apps/course_metadata/models.py (hunk 1 above)
Comment (schenedx) [reply course_discovery/apps/course_metadata/models.py]: @clintonb My understanding is that, we need to remove this from the models.py first, then create the migration file on the second PR.
See a recent example at https://github.com/edx/edx-platform/pull/15863/files and https://github.com/edx/edx-platform/pull/15911/files

Comment (clintonb) [reply course_discovery/apps/course_metadata/models.py]: Those were wrong. The committed models and migrations should always be in sync. Remove all references to the field, except the field itself first. After that, remove the field in both code and migration. See https://engineering.edx.org/django-migration-donts-f4588fd11b64.

Comment (schenedx) [reply course_discovery/apps/course_metadata/models.py]: Hmm... OK.. I'll do what you recommended, and let's see.

**Review by clintonb — APPROVED** [review]
Squash your commits.

**Pull request merged**